src/components/Sidebar/SidebarTree/TreeInfos.tsx

```diff
@@ -157,8 +156,16 @@ const TreeInfos: FC<{
   const barChartDate = currentDate < todayAt3pm ? todayAt3pm : currentDate;
 
   const wateringsSum = useMemo(() => {
-    if (!waterings || waterings.length > 0) return 0;
-    return waterings.reduce((sum, current) => sum + current.amount, 0);
+    if (!waterings) return 0;
+    const last30DaysWaterings = waterings.filter(w => {
+      const msPerDay = 86400000;
+      const elapsedMs = new Date().getTime() - Date.parse(w.timestamp);
+      return elapsedMs / msPerDay < 30;
```

Comment:
shouldn't it be `<= 30` ?
I am wondering if we count today + 29 day (==30) or if we count solely the last 30 days. Should be the same approach like for the rain

Other than that: very nice :)

Reply:
I already pushed <= 30 in the minute you submitted your review! See last commit: https://github.com/technologiestiftung/giessdenkiez-de/pull/610/commits/1887d3aad1d56fd478508125ab4a228516c6beb7

Reply:
alright, comments crossed ;)